src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -0,0 +1,137 @@
+(ns status-im.contexts.wallet.swap.setup-swap.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.contexts.wallet.common.account-switcher.view :as account-switcher]
+            [status-im.contexts.wallet.common.utils :as utils]
+            [status-im.contexts.wallet.swap.setup-swap.style :as style]
+            [utils.i18n :as i18n]
+            [utils.navigation :as navigation]
+            [utils.re-frame :as rf]))
+
+(defn- data-item
+  [{:keys [title subtitle size subtitle-icon loading?]}]
+  [quo/data-item
+   {:container-style style/detail-item
+    :blur?           false
+    :card?           false
+    :subtitle-type   (if subtitle-icon :editable :default)
+    :status          (if loading? :loading :default)
+    :title           title
+    :subtitle        subtitle
+    :size            size
+    :icon            subtitle-icon}])
+
+(defn- transaction-details
+  [{:keys [max-slippage native-currency-symbol loading-fees?]}]
+  (let [native-token (when native-currency-symbol
+                       (rf/sub [:wallet/token-by-symbol
+                                native-currency-symbol]))
+        max-fees     (rf/sub [:wallet/wallet-send-fee-fiat-formatted
+                              native-token])]
+    [rn/view {:style style/details-container}
+     [:<>
+      [data-item
+       {:title    (i18n/label :t/max-fees)
+        :subtitle max-fees
+        :loading? loading-fees?
+        :size     :small}]
+      [data-item
+       {:title         (i18n/label :t/max-slippage)
+        :subtitle      max-slippage
+        :subtitle-icon :i/edit
+        :loading?      loading-fees?}]]]))
+
+(defn view
+  []
+  (let [[pay-value set-pay-value]        (rn/use-state "")
+        {:keys [color]}                  (rf/sub [:wallet/current-viewing-account])
+        swap-transaction-data            (rf/sub [:wallet/swap])
+        currency                         (rf/sub [:profile/currency])
+        currency-symbol                  (rf/sub [:profile/currency-symbol])
+        asset-to-pay                     (rf/sub [:wallet/swap-asset-to-pay])
+        asset-to-receive                 (rf/sub [:wallet/swap-asset-to-receive])
+
+        {:keys [max-slippage swap-proposal loading-fees?
+                receive-amount network]} swap-transaction-data
```

Comment:
can deconstruct directly at the subscription

Reply:
Will do 👍 !